sign-plugin/src/main/resources/plugin.yml

```diff
@@ -0,0 +1,13 @@
+name: SignSystem
+version: ${sign.version}
+author: TheMeinerLP
+website: https://cloudnetservice.eu/
+description: CloudNet Sign Interface for the Central server
+
+main: eu.cloudnetservice.sign.plugin.ProjectMain
+
+softdepend: [CloudNetAPI]
```

Comment:
With how this plugin is made, this is NOT a soft dependency, it is very much hard.